cloudferrylib/os/compute/cold_evacuate.py

```diff
@@ -0,0 +1,354 @@
+import cfglib
+import contextlib
+import time
+import os
+from Crypto.PublicKey import RSA
```

Comment:
This seem to be a non-standard  third-party library, please add dependency to `requirements.txt`